core/toolbox.js

```diff
@@ -273,7 +293,7 @@ Blockly.Toolbox.prototype.getClientRect = function() {
   var x = toolboxRect.left;
   var y = toolboxRect.top;
   var width = this.getWidth();
-  var height = toolboxRect.height;
+  var height = toolboxRect.height + this.flyout_.height_;
```

Comment:
Please explain this change and the change on line 289.

Reply:
I've made a comment in the code explaining that this lets us correctly calculate the delete area for the horizontal toolbox.